stable/airflow/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -309,9 +309,12 @@ spec:
         {{- if or (.Values.scheduler.connections) (.Values.scheduler.existingSecretConnections) }}
         - name: connections
           secret:
-            secretName: {{ if .Values.scheduler.existingSecretConnections }}{{ .Values.scheduler.existingSecretConnections }}
-                        {{- else }}{{ include "airflow.fullname" . }}-connections
-                        {{- end }}
+            secretName: |
```

Comment:
Hi. I think this part is causing me an issue when trying to load an existing secret.
helm appends a new line at the end and now the pod is looking for the secret "airflow-connections\n"
Please have a look

Reply:
@yoavo-datricks I have raised #23858 to fix this

Reply:
@yoavo-datricks this should be fixed in `7.11.3`

Reply:
@thesuperzapper It's working, thanks!
BTW, using this script to create an initial admin user as well, super useful!